label_studio/data_import/api.py

```diff
@@ -459,8 +460,52 @@ def get(self, *args, **kwargs):
         file = file_upload.file
         if file.storage.exists(file.name):
             content_type, encoding = mimetypes.guess_type(str(file.name))
-            if content_type not in ['image/jpeg', 'image/gif', 'image/png', 'application/pdf']:
-                content_type = 'text/plain'
+            content_type = content_type or 'application/octet-stream'
+            mime_type_filter = [
+                    'image/svg+xml',
+                    # 'application/xhtml+xml',
+                    # 'text/html',
+                    # 'text/javascript'
+            ]
+            if content_type in mime_type_filter:
+                return RangedFileResponse(
```

Comment:
I would make all checks on the import step and keep this place untouched.

Reply:
personally, I believe this is the better approach. This way we don't overwrite users uploaded files, which can cause confusion. However i have moved these changes to happen on the file intake process as requested. cc @makseq

Reply:
> This way we don't overwrite users uploaded files

yeah, it's a good point I missed. However, I have taken into account the performance: every time GET API for uploaded file has to clean the SVG. So, let's wrap import with cleanup into `SVG_SECURITY_CLEANUP` env variable. If some users have troubles, they can disable it.

Reply:
>  have taken into account the performance: every time GET API for uploaded file has to clean the SVG

yes, but only SVG files and this is a fast process, not a performance issue for label studio. 

I'll add the env var